src/EditorFeatures/Core/Implementation/DocumentationComments/AbstractDocumentationCommentCommandHandler.cs

```diff
@@ -324,64 +326,101 @@ private void AddLineBreaks(SourceText text, IList<string> lines)
         }
 
         private bool TryGenerateExteriorTriviaAfterEnter(
-            SyntaxTree syntaxTree,
-            SourceText text,
-            int position,
+            Document newDocument,
+            Document originalDocument,
+            int newPosition,
             int originalPosition,
             ITextBuffer subjectBuffer,
             ITextView textView,
             DocumentOptionSet options,
             CancellationToken cancellationToken)
         {
-            // Find the documentation comment before the new line that was just pressed
-            var token = GetTokenToRight(syntaxTree, originalPosition, cancellationToken);
-            if (!IsDocCommentNewLine(token) || token.SpanStart != originalPosition)
```

Comment:
:question: Would the following change allow us to continue using just the new text here?

```csharp
var token = GetTokenToLeft(syntaxTree, newPosition, cancellationToken);
if (!IsDocCommentNewLine(token) || token.SpanStop != newPosition))
```